src/ralph/export_to_ng/resources.py

```diff
@@ -42,20 +45,67 @@ class Meta:
             'source',
             'status',
             'task_url',
+            'imei',
+            'warehouse',
+            'owner',
+            'user',
+            'remarks',
+            'service_env',
         ]
         model = models_assets.Asset
 
     def get_queryset(self):
         return self.Meta.model.objects.filter(
             type=models_assets.AssetType.back_office,
-        )
+            part_info=None,
+        ).select_related('office_info')
+
+    def dehydrate_imei(self, asset):
+        try:
+            imei = asset.office_info.imei or ''
+        except AttributeError:
+            imei = ''
+        return imei
+
+    def dehydrate_barcode(self, asset):
+        return asset.barcode or ''
+
+    def dehydrate_sn(self, asset):
+        return asset.sn or ''
+
+    def dehydrate_user(self, asset):
+        try:
+            username = asset.user.username or ''
+        except AttributeError:
+            username = ''
+        return username
+
+    def dehydrate_owner(self, asset):
+        try:
+            username = asset.owner.username or ''
+        except AttributeError:
+            username = ''
+        return username
+
+    def dehydrate_hostname(self, asset):
+        try:
+            hostname = asset.device_info.ralph_device.name or ''
+        except AttributeError:
+            hostname = ''
+        return hostname
+
+    def dehydrate_service_env(self, asset):
```

Comment:
as discussed, if we'll find it necessary, this will be handled in separated issue/PR for DC and BO at the same time.